crates/common/certificate/src/parse_root_certificate.rs

```diff
@@ -0,0 +1,132 @@
+use rustls_0_19::{
+    internal::pemfile::{certs, pkcs8_private_keys, rsa_private_keys},
+    ClientConfig,
+};
+use std::{fs, fs::File, io::BufReader, path::PathBuf};
+
+use crate::CertificateError;
+use std::io::{Error, ErrorKind};
+
+pub fn create_client_config() -> rustls_0_19::ClientConfig {
+    ClientConfig::new()
+}
```

Comment:
The role of this function is not clear. Having a better name would help a lot! Why not `create_tls_config`?

Reply:
Yes, make sense. I renamed it to `create_tls_config`